.github/workflows/docker-build.yml

```diff
@@ -3,34 +3,87 @@ name: Build & Publish to Registry
 on:
   push:
     branches:
-      - master
-  schedule:
-    - cron: '0 2 * * 0'
+      - update/new-net-version
+
+env:
+  REGISTRY: ghcr.io
+  ORGANISATION_NAME: whiteducksoftware
 
 jobs:
   update:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-
-    - name: Login to DockerHub
-      uses: docker/login-action@v1
-      with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-
-    - name: Build and push WebApp
-      uses: docker/build-push-action@v2
-      with:
-        push: true
-        context: ./src/WebApp
-        file: ./src/WebApp/Dockerfile
-        tags: whiteduck/sample-mvc:latest
-
-    - name: Build and push API
-      uses: docker/build-push-action@v2
-      with:
-        push: true
-        context: ./src/FredApi
-        file: ./src/FredApi/Dockerfile
-        tags: whiteduck/sample-api:latest
+      - uses: actions/checkout@master
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
@swolg Kannst du die Pipeline bitte noch auf buildx umstellen. Details hier: https://github.com/nmeisenzahl/grpc-health/blob/39d379de933543da8eb4615a64928aa0e2debbab/.github/workflows/build-and-push.yaml#L17

Reply:
Hab die Pipeline auf buildx umgestellt.